cfgov/unprocessed/css/print.less

```diff
@@ -28,25 +28,37 @@
     - cfgov-print
 */
 
+.respond-to-print({
+    a::after {
```

Comment:
Are we moving to using double colons now? Should we do this everywhere?

Reply:
I admit to never feeling like I fully understood pseudostuff, but I think it's used here because it's creating a pseudo_element_ within the `a`, as opposed to something like `:hover` or `:focus` that is a pseudo_class_ on the element?

I didn't actually give it any thought; the example code I found used the double colon.

Reply:
It just comes down to if we want IE8 to recognize this style or not. If we're past that, this is fine, if not, it should be `a:after`.

Reply:
Though I guess that would only matter for a print preview, since this is print styles, so double is probably the way to go.

Reply:
Ah, I still have some browser testing to do, but forgot that IE8 wouldn't recognize it. Probably worth changing, then.

Reply:
Done in 831968d